packages/graphql/src/index.ts

```diff
@@ -69,22 +70,88 @@ export async function request<T = any, V extends object = {}>(
   const { fetcher = fetch, variables = {}, headers = {}, method = "POST" } = options;
   const query_ = typeof query == "string" ? query : print(query);
 
-  return fetcher(url, {
-    ...options,
-    method,
-    body: JSON.stringify({ query: query_, variables }),
-    headers: {
-      "content-type": "application/json",
-      ...headers
-    }
-  })
+  const fetched = options.multipart ?
+    fetcher(url, {
+      ...options,
+      method: 'POST',
+      body: makeMultipartBody(query_, variables),
+      headers: {
+        "content-type": "multipart/form-data",
+        ...headers
+      }
+    }) :
+    fetcher(url, {
+      ...options,
+      method,
+      body: JSON.stringify({ query: query_, variables }),
+      headers: {
+        "content-type": "application/json",
+        ...headers
+      }
```

Comment:
> If only three fields are different, only the options object could be modified. Not that it matters, but probably less js to send.

It is OK for me. Usually I prefer to aggregate repeated conditions (`options.multipart ?`) in a single `if`, to avoid missing one in case of changes but I don't have a strong opinion.